internal/config/auth_config_test.go

```diff
@@ -311,22 +319,45 @@ func TestLoginAddsHostIfNotAlreadyAdded(t *testing.T) {
 	require.Contains(t, hosts, "github.com")
 }
 
+// This test mimics the behaviour of logging in with a token and not providing
+// a git protocol.
+func TestLoginAddsUserToConfigWithoutGitProtocolOrSecureStorage(t *testing.T) {
+	// Given we are not logged in
+	authCfg := newTestAuthConfig(t)
+
+	// When we log in without git protocol or secure storage
```

Comment:
This comment doesn't seem to be correct with regards to secure storage. I think we do want secure storage (to avoid the oauth token writing the full structure) but I just think the test name and this comment is incorrect.